ios/Gekidou/Sources/Gekidou/Storage/Database+Mentions.swift

```diff
@@ -28,11 +28,12 @@ extension Database {
 
     public func getThreadMentions(_ db: Connection) -> Int {
         let stmtString = """
-        SELECT SUM(unread_mentions) \
-        FROM Thread t
-        INNER JOIN Post p ON t.id=p.id \
-        INNER JOIN Channel c ON p.channel_id=c.id
-        WHERE c.delete_at = 0
```

Comment:
this query was using a plan that visited every record of the post table, which can be massive causing a hang.